### ISSUE TITLE
In cec code key_action up and down reversed

### ISSUE THREAD

**Issue by wkewlin** [irtimmer/moonlight-embedded#156]
At least on my Panasonic TV cec sends duration == 0 for key press and duration != 0 for release. This is probably always true, seems at least reasonable.
So input.c need the following change:

```
  if (value != 0) {                                                                                                            
     short code = 0x80 << 8 | keyCodes[value];                                                                                  
-    LiSendKeyboardEvent(code, (key.duration > 0)?KEY_ACTION_DOWN:KEY_ACTION_UP, 0);                                            
+    LiSendKeyboardEvent(code, (key.duration > 0)?KEY_ACTION_UP:KEY_ACTION_DOWN, 0);                                            
   }     
```

**Comment by schulzad** [comment]
I can test this on my Panasonic TV as well.

**Comment by cgutman** [comment]
libcec-daemon source also indicates duration == 0 is down and duration != 0 is up.

https://github.com/bramp/libcec-daemon/blob/583653bd3668b2797ebf057d20dfa0ecbc329b5a/src/main.cpp#L324

@irtimmer Can you make the above change?

**Comment by irtimmer** [comment]
Fixed with c10195f524c1f2856dfd5b9a043ee702978bd8b5
